### ISSUE TITLE
Compile error on macos in R3BOnlineSpectraLosStandalone

### ISSUE THREAD

**Issue by ChristianTackeGSI** [R3BRootGroup/R3BRoot#404]
Hi,

one of our last CI [runs](https://cdash.gsi.de/testDetails.php?test=8123155&build=269932) found an issue here:

```console
/Users/alfaci/alfaci/workspace/FairRootGroup_FairSoft_PR-397/build/test_workdir/stage/spack-stage-r3broot-develop-myzse2azudm6cwj57nxbg2xc4qdbqo33/spack-src/r3bbase/R3BOnlineSpectraLosStandalone.cxx:453:18: error: variable-sized object may not be initialized
    Int_t nPartc[fNofLosDetectors] = { 0 };
                 ^~~~~~~~~~~~~~~~
```

Could you look into this?

Maybe consider something like this?
```c++
std::array<Int_t, fNofLosDetectors> nPartc{0};
```

**Comment by hapol** [comment]
The error:
```
/Users/hapol/CODES/R3BROOT/R3BRoot/r3bbase/R3BOnlineSpectraLosVsSci2.cxx:852:18: error: variable-sized object may not be initialized
    Int_t nPartc[fNofLosDetectors] = { 0 };
                 ^~~~~~~~~~~~~~~~
1 warning and 1 error generated.
```
prevents the present dev version to be compiled with Apple clang version 11.0.3 (clang-1103.0.32.62).
Could you please  @jose-luis-rs take a look at it?

**Comment by jose-luis-rs** [comment]
I have solved the problem again.

**Comment by kresan** [comment]
fixed with #410. Closing

**Comment by ChristianTackeGSI** [comment]
Thanks everybody for taking care!
